bokeh/client/connection.py

```diff
@@ -388,14 +388,11 @@ def have_reply_or_disconnected() -> bool:
 
     def _send_patch_document(self, session_id: ID, event: DocumentChangedEvent) -> None:
         # XXX This will cause the client to always send all columns when a CDS
-        # is mutated in place. Additionally we set use_buffers=False below as
-        # well, to suppress using the binary array transport. Real Bokeh server
-        # apps running inside a server can handle these updates much more
-        # efficiently
+        # is mutated in place.
         from bokeh.document.events import ColumnDataChangedEvent
         if isinstance(event, ColumnDataChangedEvent):
             event.cols = None
```

Comment:
@bryevdv, what's the point of this? The comment doesn't explain anything.

Reply:
Buffers originally only worked from server to browser, and not from client session to browser. The real bokeh apps was in reference to the practice (at the time) of some users trying to make apps “outside” the server, with session.loop_until_closed

Reply:
So, I can remove this case as well?